### PR TITLE
MM-31250 Implement export command

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://github.com/mattermost/mmctl/pull/296

Updated:
- Self-Managed Admin Guide > Administration > mmctl Command Line Tool (Beta)
   - Added ``mmctl export`` and its child commands

### CODE REVIEW

**Review by isacikgoz — APPROVED** [review]
LGTM 👍 Adding @streamer45 as a reviewer since he implemented this.

**Review by streamer45 — APPROVED** [review]
Thank you @cwarnermm , looks good :+1:

File: source/administration/mmctl-cli-tool.rst [hunk 1]
```diff
@@ -23,6 +23,7 @@ This feature was developed to a large extent by community contributions and we'd
    - `mmctl completion`_ - Generates autocompletion scripts for bash and zsh
    - `mmctl config`_ - Configuration Management
```
Comment (justinegeffen) [inline source/administration/mmctl-cli-tool.rst:24]:
```suggestion
   - `mmctl config`_ - Configuration management
```

File: source/administration/mmctl-cli-tool.rst [hunk 2]
```diff
@@ -23,6 +23,7 @@ This feature was developed to a large extent by community contributions and we'd
...
    - `mmctl docs`_ - Generates mmctl documentation
+   - `mmctl export`_ - Exports Management
```
Comment (justinegeffen) [inline source/administration/mmctl-cli-tool.rst:26]:
```suggestion
   - `mmctl export`_ - Exports management
```

File: source/administration/mmctl-cli-tool.rst [hunk 3]
```diff
@@ -23,6 +23,7 @@ This feature was developed to a large extent by community contributions and we'd
...
    - `mmctl group`_ - Group Management
```
Comment (justinegeffen) [inline source/administration/mmctl-cli-tool.rst:27]:
```suggestion
   - `mmctl group`_ - Group management
```

File: source/administration/mmctl-cli-tool.rst [hunk 4]
```diff
@@ -23,6 +23,7 @@ This feature was developed to a large extent by community contributions and we'd
...
    - `mmctl integrity`_ - Database record integrity
    - `mmctl ldap`_ - LDAP Management
```
Comment (justinegeffen) [inline source/administration/mmctl-cli-tool.rst:29]:
```suggestion
   - `mmctl ldap`_ - LDAP management
```

File: source/administration/mmctl-cli-tool.rst [hunk 5]
```diff
@@ -1950,6 +1951,263 @@ mmctl docs
...
+
+.. code-block:: sh
+
+  mmctl export download [exportname] [filepath] [flags]
+
+**Example**
+
+.. code-block:: sh
+
+  # you can indicate the name of the export and its destination path
```
Comment (justinegeffen) [inline source/administration/mmctl-cli-tool.rst:2058]:
```suggestion
  # You can indicate the name of the export and its destination path
```

File: source/administration/mmctl-cli-tool.rst [hunk 6]
```diff
@@ -1950,6 +1951,263 @@ mmctl docs
...
+   $ mmctl export download samplename sample_export.zip
+
+   # or if you only indicate the name, the path would match it
```
Comment (justinegeffen) [inline source/administration/mmctl-cli-tool.rst:2061]:
```suggestion
   # If you only indicate the name, the path will match it
```

**Review by justinegeffen — COMMENTED** [review]
Just minor suggestions, otherwise LGTM. Approving as non-blocking imo.

**Review by justinegeffen — APPROVED** [review]

File: source/administration/mmctl-cli-tool.rst [hunk 7]
```diff
@@ -21,11 +21,12 @@ This feature was developed to a large extent by community contributions and we'd
    - `mmctl channel`_ - Channel Management
```
Comment (amyblais) [inline source/administration/mmctl-cli-tool.rst:21]: This doc page appears "broken" when clicking View file to get the preview.

Comment (cwarnermm) [reply source/administration/mmctl-cli-tool.rst:21]: @amyblais I built the documentation locally without issues to verify.

**Pull request merged**